backend/src/managers/UserManager.ts

```diff
@@ -74,7 +74,7 @@ export default class UserManager {
         return user;
     }
 
-    async getFullProfile(username: string, forUserId: number): Promise<UserProfile | undefined> {
+    async getFullProfile(username: string, forUserId: number): Promise<UserInfo | undefined> {
```

Comment:
minor: maybe rename it to `getFullUserInfo` if return type is changed from `UserProfile` to `UserInfo`?

Reply:
Renamed to `getByUsernameWithVote` (also fixed some caching issues).